notebooks/training_guideline.ipynb

```diff
@@ -311,10 +311,7 @@
     "# In case you want to test the segmentation with a pre-trained model created using this notebook,\n",
     "# uncomment the lines below.\n",
     "#\n",
-    "#if not download_data(\"https://osf.io/u2j9t/?action=download\", \"TEM_3c_512_2018-11-10_21-32-36.zip\")==0:\n",
-    "#     print('ERROR: Data was not succesfully downloaded and unzipped - please check your link and filename and try again.')\n",
-    "#else:\n",
-    "#    print('Data downloaded and unzipped succesfully.')\n",
```

Comment:
@mathieuboudreau to simplify, i was thinking of simply adding this model inside TEM_striatum.zip-- less confusing for the users (also, in addition, if one day we decide to change the training dataset we won't forget to also update the model)

Reply:
Sure – that makes sense!